lib/fluent/plugin/in_tail.rb

```diff
@@ -120,26 +119,20 @@ def configure_tag
     def start
       super
 
+      @parser = parser_create(conf: @config.elements('parse').first)
```

Comment:
I'm sorry that I missed to point about it... `create_parser` should be called in `configure` method.

Parser plugin helper will creates instance of `parser`, and then will call `#start` of created parser automatically. No need to call `@parser.start` by plugins themselves.
https://github.com/fluent/fluentd/blob/master/lib/fluent/plugin_helper/parser.rb#L95